### PR TITLE
Feedback implementation

### PR DESCRIPTION
**Pull request by emashliles**
Implement some of the feedback from review board.

### CODE REVIEW

File: src/tic_tac_toe/game.clj [hunk 1]
```diff
@@ -40,10 +40,10 @@
...
-  (cond  (is-tie? board)  (println "Game Over - Tie.")
-         (is-win? board)  (println "Game Over - Computer Wins.")
+  (cond  (is-tie? board) (println "Game Over - Tie.")
+         (is-win? board) (println "Game Over - Computer Wins.")
          :else
   (let  [marked-board  (human-turn board "X")]
-   (cond  (is-win? marked-board)  (println "Game Over - Huamn Wins.")
-         (is-tie? marked-board)  (println "Game Over - Tie.")
-         :else  (do  (game-loop-human-computer  (computer-turn marked-board "O") "O" ))))))
+   (cond (is-win? marked-board) (println "Game Over - Huamn Wins.")
```
Comment (gemcfadyen) [inline src/tic_tac_toe/game.clj:47]: Still a typo here `Huamn`

File: src/tic_tac_toe/game.clj [hunk 2]
```diff
@@ -40,10 +40,10 @@
...
+         (is-tie? marked-board) (println "Game Over - Tie.")
+         :else  (do (game-loop-human-computer (computer-turn marked-board "O") "O" ))))))
```
Comment (gemcfadyen) [inline src/tic_tac_toe/game.clj:49]: Rather than using strings for X and O, I think you could use symbols 'X 'O throughout

File: README.md [hunk 3]
```diff
@@ -1,10 +1,32 @@
...
 
 ## Pre-requisites
 
-+ Java
-+ Leinigen
++ Java 8
++ Leinigen v2.7.1
+
+## Test
+
```
Comment (RabeaGleissner) [inline README.md:11]: Thanks for adding the instructions to the Readme! 😅

**Review by RabeaGleissner — COMMENTED** [review]
Unfortunately it looks like the CI is failing... I would recommend not submitting your code (i.e. sending the link to your PR out) until you've got the CI green.

**Pull request closed without merging**